docker/dockerfiles/fe/fe-ubuntu.Dockerfile

```diff
@@ -26,13 +26,13 @@ FROM ubuntu:22.04
 ARG STARROCKS_ROOT=/opt/starrocks
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends \
-        default-jdk mysql-client curl vim tree net-tools less tzdata linux-tools-common linux-tools-generic && \
+        openjdk-11-jdk mysql-client curl vim tree net-tools less tzdata linux-tools-common linux-tools-generic && \
         ln -fs /usr/share/zoneinfo/UTC /etc/localtime && \
         dpkg-reconfigure -f noninteractive tzdata && \
         rm -rf /var/lib/apt/lists/*
 RUN echo "export PATH=/usr/lib/linux-tools/5.15.0-60-generic:$PATH" >> /etc/bash.bashrc && \
         touch /.dockerenv
-ENV JAVA_HOME=/lib/jvm/default-java
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-arm64
```

Comment:
this is not the right one, the dockerfile should work for both x86 and arm arch. this line breaks it.